tool/teleport/common/teleport.go

```diff
@@ -1014,6 +1019,28 @@ func onIntegrationConfExternalAuditCmd(params config.IntegrationConfExternalClou
 	if err != nil {
 		return trace.Wrap(err)
 	}
+	if params.Bootstrap {
+		bootstrapClt := &externalcloudaudit.DefaultBootstrapInfraClient{
+			Glue:   glue.NewFromConfig(cfg),
+			S3:     s3.NewFromConfig(cfg),
+			Athena: athena.NewFromConfig(cfg),
+		}
+
+		err = externalcloudaudit.BootstrapInfra(ctx, bootstrapClt, &ecatypes.ExternalCloudAuditSpec{
+			SessionsRecordingsURI:  params.SessionRecordingsURI,
+			AuditEventsLongTermURI: params.AuditEventsURI,
+			AthenaResultsURI:       params.AthenaResultsURI,
+			GlueDatabase:           params.GlueDatabase,
+			GlueTable:              params.GlueTable,
+			AthenaWorkgroup:        params.AthenaWorkgroup,
+		}, params.Region)
+		if err != nil {
+			return trace.Wrap(err)
+		}
+
+		return nil // Remove this (FOR TESTING ONLY)
```

Comment:
I think you can probably go ahead and remove this from the PR (just add it back locally if you want to)

Reply:
Removed in [141fbba](https://github.com/gravitational/teleport/pull/33272/commits/141fbba3b3c7e20f25ed0572dc17c5bc8eb295f6)